src/components/Quote.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+
+import './Quete.css';
+
+function Fetch() {
+  const [data, setData] = useState([]);
```

Comment:
- [ ] [OPTIONAL] It is always a good practice to name our components the same name given to the file. This function would have been named Quotes, `function quotes`.